src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1229,23 +1229,125 @@ private int SkipBlockComment(int i)
             return i;
         }
 
-        private static char Backtick(char c)
+        private string Backtick(char c)
         {
             switch (c)
             {
-                case '0': return '\0';
-                case 'a': return '\a';
-                case 'b': return '\b';
-                case 'e': return '\u001b';
-                case 'f': return '\f';
-                case 'n': return '\n';
-                case 'r': return '\r';
-                case 't': return '\t';
-                case 'v': return '\v';
-                default: return c;
+                case '0': return "\0";
+                case 'a': return "\a";
+                case 'b': return "\b";
+                case 'e': return "\u001b";
+                case 'f': return "\f";
+                case 'n': return "\n";
+                case 'r': return "\r";
+                case 't': return "\t";
+                case 'u': return ScanUnicodeEscapeSequence();
+                case 'v': return "\v";
+                default: return c.ToString();
             }
         }
 
+        private string ScanUnicodeEscapeSequence()
+        {
+            // Unicode replacement char, used to represent unknown, unrecognized or unrepresentable characters.
+            var unknownChar = ((char)0xffdd).ToString();
```

Comment:
If a literal string is not possible, you should make this static so we create just one instance instead of creating an instance every time we scan `` `u``.